api/test/integration/env/configurations/base/manager/configuration_files/ossec_base.conf

```diff
@@ -7,7 +7,7 @@
         <port>1516</port>
         <bind_addr>0.0.0.0</bind_addr>
```

Comment:
What about this one?

Reply:
It is a configuration to be used in the master of a test environment, so it's expected.